### PR TITLE
Added manifest for HoYoverse.GenshinImpact

### PR DESCRIPTION
**Pull request by Dvd-Znf**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102719)

### CODE REVIEW

**Comment by Dvd-Znf** [comment]
@microsoft-github-policy-service agree

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+Installers:
+- InstallerUrl: https://sg-public-api.hoyoverse.com/event/download_porter/link/ys_global/genshinimpactpc/default
+  Architecture: x64
+  InstallerType: exe 
+  InstallerSwitches:
+    Silent: n/a
+    SilentWithProgress: n/a
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml:9]: These switches don't appear to give an unattended install. The install must be able to be run unattended in order for the package to be accepted

image

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+PackageLocale: en-US
+Publisher: HoYoverse
+PublisherUrl: https://www.hoyoverse.com/en-us
+PackageName: Genshin Impact Launcher
+PackageUrl: https://genshin.hoyoverse.com/en/home
+License: copyright cognosphere
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml:8]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Copyright: ©COGNOSPHERE
+ShortDescription: Genshin Impact is an action role-playing game developed and published by miHoYo.
+Tags:
+- Genshin
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml:12]:
```suggestion
- genshin
- rpg
```

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+Installers:
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.installer.yaml:3]: This application may also have a dependency, which isn't supported yet. I'm not sure if an unattended install would hide this prompt
image

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+PackageIdentifier: HoYoverse.GenshinImpact
+PackageVersion: 2.25.2.0
+PackageLocale: en-US
+Publisher: HoYoverse
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml:4]:
```suggestion
Publisher: COGNOSPHERE PTE. LTD.
```
This should match Control Panel

File: manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+PublisherUrl: https://www.hoyoverse.com/en-us
+PackageName: Genshin Impact Launcher
```
Comment (Trenly) [inline manifests/h/HoYoverse/GenshinImpact/2.25.2.0/HoYoverse.GenshinImpact.locale.en-US.yaml:6]:
```suggestion
PackageName: Genshin Impact
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
Please also include the Yaml Schema headers when updating this file. If you use a tool like wingetcreate or YamlCreate they will automatically be included

**Comment by Dvd-Znf** [comment]
Since the installer doesn't offer an unattended install i'll close the PR   
  
Previously submitted PR's for this installer have also been closed due to this issue

**Pull request closed without merging**